ctms/crud.py

```diff
@@ -932,6 +933,17 @@ def create_acoustic_newsletters_mapping(dbsession, source, destination):
     row = AcousticNewsletterMapping(source=source, destination=destination)
     # This will fail if the mapping already exists.
     dbsession.add(row)
+
+    # Force a resync of all contacts that are subscribed to this newsletter.
```

Comment:
It feels as though this should be in a separate CRUD function, (maybe `sync_newsletter_with_acoustic`) since the management of newsletter mappings and when they're synced seems like separate concerns.

But, it does make sense to attempt to sync a newsletter just after it's added. 

The dream code version of a management CLI (to me) seems like it'd be:
```
add <newsletter name>:<sub_ column>
add <newsletter name>:<sub_ column> --sync
sync <newsletter name>
```

Reply:
Yes, I agree.

Let's ditch this command and add an `acoustic` CLI with subcommands in #569